app.ts

```diff
@@ -0,0 +1,40 @@
+/// <reference path="./typings/tsd.d.ts" />
+
+import * as express from "express";
+import * as bodyParser from "body-parser";
+import * as errorHandler from "errorhandler";
+
+import {HomeController} from "./src/controller/home/homeController";
+import {serviceContainer} from "./src/config/service-container";
+
+let app = express();
+
+app.set("views", __dirname + "/src/views");
+app.set("view engine", "twig");
+app.set("view options", {layout: false});
+app.use(bodyParser.urlencoded({ extended: true }));
+app.use(bodyParser.json());
+app.use(express.static(__dirname + "/src/media"));
+
+let env = process.env.NODE_ENV || "development";
+
+if (env === "development") {
+    app.use(errorHandler());
+}
+
+app.get("/", (req, res) => {
+    <HomeController>serviceContainer.getService("homeController").index(req, res);
```

Comment:
There is probably a really good DI container out there - for now this is ok and does what it needs to do.